### PR TITLE
MGMT-2583: Add warning about wiping disks

### PR DESCRIPTION
**Pull request by mareklibra**

### CODE REVIEW

File: src/components/clusterConfiguration/FormatDiskWarning.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Text } from '@patternfly/react-core';
+import { ExclamationTriangleIcon } from '@patternfly/react-icons';
+import { global_warning_color_100 as warningColor } from '@patternfly/react-tokens';
+
+const FormatDiskWarning = () => (
+  <Text component="p">
+    <ExclamationTriangleIcon className="status-icon" color={warningColor.value} size="sm" />
+    &nbsp;Note that all disks will be formatted when the installation starts.
```
Comment (rollandf) [inline src/components/clusterConfiguration/FormatDiskWarning.tsx:9]: We don't format all disks. We are deleting the file system on bootable disks.

**Comment by mareklibra** [comment]
Marking as WIP due to ongoing discussion - https://coreos.slack.com/archives/CUPJTHQ5P/p1604408618363600

**Review by andybraren — APPROVED** [review]
lgtm

**Comment by mareklibra** [comment]
Backend patch is ready, ublocking here

**Review by jtomasek — APPROVED** [review]

File: src/components/clusterConfiguration/FormatDiskWarning.tsx (hunk 1 above)
Comment (nirfarkas) [reply src/components/clusterConfiguration/FormatDiskWarning.tsx]: So, a more accurate text should be: "Note that all bootable disks will be formatted when the installation starts"?

Comment (andybraren) [reply src/components/clusterConfiguration/FormatDiskWarning.tsx]: `Note that bootable disks will be formatted when the installation starts.` is what merged in this PR, but adding `all` to that sentence seems like a good tweak for extra clarity. I've created #291 if you agree @rollandf.

**Pull request merged**